### PR TITLE
Resolved Mailman database service connect errors in docker

### PR DESCRIPTION
**Pull request by namangupta01**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by namangupta01** [comment]
Previous discussion at #2899

**Comment by jywarren** [comment]
Great!!! Is this ready to merge?

**Comment by namangupta01** [comment]
There is some codeclimate error i will push a commit to remove one of them (and other error is to be ignore --i think) then it is good to go.

**Comment by jywarren** [comment]
OK cool - what about #2888 - also to be merged with this?

**Comment by namangupta01** [comment]
Let's first merge this and check on stable? The we can merge? We can do
either way!

**Comment by jywarren** [comment]
Want me to just resolve the codeclimate issues?

**Comment by namangupta01** [comment]
Just commit minor changes to remove one codeclimate issue please resolve the second codeclimate issue.

**Comment by namangupta01** [comment]
@jywarren One of the codeclimate issue is fixed. Please adjust the second issue.

**Comment by jywarren** [comment]
Done!

File: script/mailman_server [hunk 1]
```diff
@@ -9,9 +9,10 @@ Mailman.config.ignore_stdin = true #ignore input from STDIN
 
 Mailman.config.poll_interval = 5 # To poll for mails in every 5 seconds
 
+
 # Email Configration for mailman
 Mailman.config.pop3 = {
-  server: ENV['SERVER_ADDRESS'], # example pop.example.com 
+  server: 'pop.gmail.com', # example pop.example.com
```
Comment (jywarren) [inline script/mailman_server]: Ah, shouldn't we switch this back to an env variable? Maybe in #2888?

**Comment by namangupta01** [comment]
Yeah. I forgot to change it after pushing to unstable. Will do in #2888

**Comment by namangupta01** [comment]
@jywarren Have you pushed it to stable? So that we could check it.

**Comment by jywarren** [comment]
And it worked!!!!!
https://stable.publiclab.org/notes/bkleist/04-04-2017/uwec-air-quality-monitoring#c17376

**Comment by namangupta01** [comment]
🎉  🎉

**Comment by namangupta01** [comment]
:tada: :tada:

**Comment by jywarren** [comment]
Hi, Naman! I just deployed to production and I tried emailing to `notifications@publiclab.org` for these two:

https://publiclab.org/questions/warren/06-26-2018/are-there-lists-of-plants-and-the-chemicals-they-can-remediate

https://publiclab.org/notes/Delaney/06-25-2018/africatown-connections-blueway-celebration

But I didn't see them come through! Maybe we've forgotten something?

**Comment by jywarren** [comment]
wait, i think i found the issue!

**Comment by jywarren** [comment]
Fixed it! I had the email address wrong. 

I deleted the comment, but it def. works! Pretty amazing!

screen shot 2018-06-26 at 7 05 05 pm

**Pull request merged**